FND/Agreements/Contracts.rdf

```diff
@@ -287,30 +296,29 @@ A unilateral contract differs from a Bilateral Contract, in which the parties ex
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>written contract</rdfs:label>
-		<skos:definition>a formal contract that is written and signed by the parties thereto</skos:definition>
+		<skos:definition>formal contract that is written and signed by the parties thereto</skos:definition>
 	</owl:Class>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;definesTermsFor">
 		<rdfs:label>defines terms for</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
-		<skos:changeNote>added to support requirements in Business Entities, 2013-05-22</skos:changeNote>
-		<skos:definition>the contract sets out the terms for the something</skos:definition>
+		<skos:definition>relates a contract to something for which the contract defines legally binding terms and conditions</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasContractParty">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
 		<rdfs:label>has contract party</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
 		<rdfs:range rdf:resource="&fibo-fnd-agr-ctr;ContractParty"/>
-		<skos:definition>has a party which is a signatory to the contract and to which is granted certain rights and obligations as defined in the contract and which concedes certain rights to and imposes certain obligations upon the other party as defined in the contract</skos:definition>
+		<skos:definition>indicates a party that is a signatory to a contract, to which certain rights and obligations are granted, and that concedes certain rights to and imposes certain obligations on another party as specified</skos:definition>
```

Comment:
Inconsistent with definition for PromissoryNote which says that both parties do not need to sign it.

Reply:
I'll reword it and remove the requirement that someone needs to be a signatory to be a party to the contract - that was in the original ontology, however.

Reply:
Done - reworded - please review the latest definition